code/components/gta-net-five/src/MumbleVoice.cpp

```diff
@@ -165,6 +165,9 @@ static struct
 	volatile int nextConnectDelay;
 	volatile uint64_t nextConnectAt;
 
+	std::string overrideAddress;
```

Comment:
Please store this as a net::PeerAddress directly so resolution cost/failures can be handled at native call time.